frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditCheckbox/EditCheckbox.tsx

```diff
@@ -103,6 +104,17 @@ export const EditCheckbox = ({ field }: EditCheckboxProps): JSX.Element => {
 
   const watchedInputs = watch()
 
+  const optionsValidation = useCallback(
+    (opts: string) => {
+      const textareaValidation = SPLIT_TEXTAREA_VALIDATION.validate(opts)
+      if (textareaValidation !== true) return textareaValidation
```

Comment:
```suggestion
      if (!textareaValidation) return textareaValidation
```

Reply:
added comment for clarification in [36b7ef3](https://github.com/opengovsg/FormSG/pull/5319/commits/36b7ef397d0882ba7c4ed6a553aced701e8b4b00) !